sda-commons-shared-otel-agent/build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins{
+  id "com.github.johnrengelman.shadow" version "7.1.2"
+}
+ext {
+  otelAgentVersion = '1.13.1'
+  otelAgentAlphaVersion = '1.13.1-alpha'
```

Comment:
* does the version need to match the open telemetry version defined in the dependencies module?
* I would also move these versions to the dependencies module

Reply:
They are supposed to converge eventually, but can be different. The agent has the opentelemetry-api as a dependency and sometimes  takes a while to be updated.
I will move them.